Configurations/unix-Makefile.tmpl

```diff
@@ -654,9 +654,11 @@ errors:
 	( cd $(SRCDIR); $(PERL) util/ck_errf.pl -strict */*.c */*/*.c )
 	( cd $(SRCDIR); $(PERL) util/mkerr.pl -internal )
 	( cd $(SRCDIR)/engines; \
-	  for e in *.ec; do \
-	      $(PERL) ../util/mkerr.pl -conf $$e -rebuild *.c; \
-	  done )
+          $(PERL) ../util/mkerr.pl -conf e_capi.ec e_capi.c ; \
+          $(PERL) ../util/mkerr.pl -conf e_dasync.ec e_dasync.c ; \
+          $(PERL) ../util/mkerr.pl -conf e_ossltest.ec e_ossltest.c ; \
+          cd afalg; \
+          $(PERL) ../../util/mkerr.pl -conf e_afalg.ec e_afalg.c )
```

Comment:
Except that the previous lines were wrong; they never rescanned source files looking for new errors or function codes. :)

I gave up on trying to move afalg.  Hopefully you or @mattcaswell could do it.